TermTk/TTkWidgets/progressbar.py

```diff
@@ -0,0 +1,148 @@
+# MIT License
+#
+# Copyright (c) 2022 Eugenio Parodi <ceccopierangiolieugenio AT googlemail DOT com>
+#                    Luchr          <https://github.com/luchr>
```

Comment:
Feel free to add your name/e-mail

Reply:
The way you've written it, fits perfectly.